packages/tasks/src/pipelines.ts

```diff
@@ -658,6 +658,12 @@ export const PIPELINE_DATA = {
 		color: "blue",
 		hideInDatasets: true,
 	},
+	"keypoint-detection": {
+		name: "Keypoint Detection",
+		modality: "cv",
+		color: "red",
+		hideInDatasets: true,
+	},
```

Comment:
Should it be a subtask of `object-detection` instead? There's already a `face-detection` subtask, for example.

Reply:
I think it's not. object detection is where you detect bounding boxes, but in this one outputs look like these: 
![keypoint-detection-output](https://github.com/user-attachments/assets/ce31b67b-314c-496c-a399-47db2bcc1e7b)
<img width="494" alt="Screenshot 2024-08-29 at 14 01 01" src="https://github.com/user-attachments/assets/98a5cb24-05f9-434e-b2e9-ce02c3cd97f8">